sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c

```diff
@@ -280,9 +266,12 @@ static void test_noexec_rodata(void) {
  * Test that the .data section in RAM is not executable.
  */
 static void test_noexec_rwdata(void) {
-  if (!sram_exec_enable(TOP_EARLGREY_SRAM_CTRL_MAIN_REGS_BASE_ADDR)) {
-    base_printf("failed to enable main RAM execution\n");
-  }
+  dif_sram_ctrl_t sram_ctrl;
+  CHECK(dif_sram_ctrl_init(
```

Comment:
```suggestion
  CHECK_DIF_OK(dif_sram_ctrl_init(
```

Reply:
applies throughout

Reply:
Ah, my apologies I missed that.

Reply:
No worries, it doesn't show up in the diff.